App/features/account/actions.ts

```diff
@@ -86,25 +86,3 @@ export const setRecoveryPhrase = createAction(
     return (recoveryPhrase: string) => resolve({ recoveryPhrase })
   }
 )
-
-export const getCafeSessionsRequest = createAction(
-  'account/GET_CAFE_SESSIONS_REQUEST'
-)
-
-export const refreshCafeSessionsRequest = createAction(
-  'account/REFRESH_CAFE_SESSIONS_REQUEST'
-)
-
-export const cafeSessionsSuccess = createAction(
-  'account/CAFE_SESSIONS_SUCCESS',
-  resolve => {
-    return (sessions: ReadonlyArray<ICafeSession>) => resolve({ sessions })
-  }
-)
-
-export const cafeSessionsError = createAction(
-  'account/CAFE_SESSIONS_ERROR',
-  resolve => {
-    return (error: any) => resolve({ error })
-  }
-)
```

Comment:
Moving this cafe-related redux out of the `account` redux and into the `cafes` redux feature.